itest/send_test.go

```diff
@@ -754,6 +754,10 @@ func testReattemptFailedReceiveUniCourier(t *harnessTest) {
 	// proof(s).
 	t.Logf("Attempting to confirm asset received by receiver node")
 	AssertNonInteractiveRecvComplete(t.t, receiveTapd, 1)
+
+	// Confirm that the sender tapd node eventually receives the asset
+	// transfer and publishes an asset recv complete event.
+	assertAssetRecvCompleteEvent(t, ctxb, 5*time.Second, eventNtfns)
```

Comment:
Maybe could go a tiny bit further and also check receiving address?

Reply:
Good idea! I've added the recv address as an arg to `assertAssetRecvCompleteEvent`.